posts/cuda-aware-mpi-example/src/Makefile

```diff
@@ -41,7 +41,10 @@ GENCODE_SM35    := -gencode arch=compute_35,code=sm_35
 GENCODE_SM35    := -gencode arch=compute_37,code=sm_37
 GENCODE_SM50    := -gencode arch=compute_50,code=sm_50
 GENCODE_SM52    := -gencode arch=compute_52,code=\"sm_52,compute_52\"
-GENCODE_FLAGS   := $(GENCODE_SM20) $(GENCODE_SM30) $(GENCODE_SM35) $(GENCODE_SM37) $(GENCODE_SM50) $(GENCODE_SM52)
+GENCODE_SM60    := -gencode arch=compute_60,code=sm_60
+GENCODE_FLAGS   := $(GENCODE_SM20) $(GENCODE_SM30) $(GENCODE_SM35) 	\
+			$(GENCODE_SM37) $(GENCODE_SM50) $(GENCODE_SM52) \
+			$(GENCODE_SM60)
```

Comment:
For Volta CUDA 9 compatibility please change to:
GENCODE_FLAGS   := $(GENCODE_SM30) $(GENCODE_SM35) $(GENCODE_SM37) $(GENCODE_SM50) $(GENCODE_SM52) $(GENCODE_SM60) $(GENCODE_SM70)